Gordon360/Services/ProfileService.cs

```diff
@@ -126,10 +126,6 @@ public IEnumerable<AdvisorViewModel> GetAdvisors(string id)
         public CliftonStrengthsViewModel GetCliftonStrengths(int id)
         {
             var strengths = _unitOfWork.CliftonStrengthsRepository.FirstOrDefault(x => x.ID_NUM == id);
-            if(strengths == null)
-            {
-                return null;
-            }
             return strengths;
```

Comment:
Might as well return directly instead of putting it in an intermediate variable right?
```C#
return unitOfWork.CliftonStrengthsRepository.FirstOrDefault(x => x.ID_NUM == id);
```